### PR TITLE
LPS-162580 KB templates custom empty state

### PR DESCRIPTION
**Pull request by boton**
### Before
image

### After

image

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBTemplatesManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -126,6 +126,30 @@ public CreationMenu getCreationMenu() {
...
+			dropdownItem -> {
+				dropdownItem.setHref(
+					PortletURLBuilder.createRenderURL(
+						_liferayPortletResponse
+					).setMVCPath(
+						"/admin/common/edit_kb_template.jsp"
+					).setRedirect(
+						PortalUtil.getCurrentURL(_httpServletRequest)
+					).buildPortletURL());
+
```
Comment (adolfopa) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBTemplatesManagementToolbarDisplayContext.java:146]: Small thing: you need to remove this blank line.

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBTemplatesManagementToolbarDisplayContext.java (hunk 1 above)
Comment (boton) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBTemplatesManagementToolbarDisplayContext.java]: done [48f0ad3](https://github.com/liferay-lima/liferay-portal/pull/3481/commits/48f0ad33f998128faee953b55d366399e3f22054)

**Comment by boton** [comment]
Sorry, @adolfopa I re-request review by mistake :(

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

**Review by ambrinchaudhary — COMMENTED** [review]
Hey @boton, shouldn't be both messages (from the navigation pane and center pane) the same? One is "no templates were found" and the other "there are no article templates". 
Otherwise LGTM

**Comment by boton** [comment]
> Hey @boton, shouldn't be both messages (from the navigation pane and center pane) the same? One is "no templates were found" and the other "there are no article templates". Otherwise LGTM

Done

image

Update the description too.

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:stop

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**